### DISCUSSION TITLE
Organize Command

### DISCUSSION THREAD

**Discussion by KohlrabitoDev** [Kwantux/networks#53] [Ideas]
A command like /net organize or something to automatically sort your items even in misc. chests. This would go through & re-stack any items that can be stacked or at least place similar items next to each other. (E.g. I'm tired of having 7 diamonds in chest 1, then having 3 diamonds in chest 4... It would be nice for the input chest to just place them all in chest 1... Also having 3 different types of wood in different chests kinda sucks.)

**Comment by Kwantux** [comment]
This is a good point. However I'm not sure about the way of implementation..
For now, I'll make it a command, but for the future of this plugin, I planned to slowly move away from commands and integrate more into vanilla.

>  I'm tired of having 7 diamonds in chest 1, then having 3 diamonds in chest 4... It would be nice for the input chest to just place them all in chest 1...

That's how the plugin already works right now. One chest is always prioritized over the others.
You can manually define a chest's priority, but if you don't it's always the order you placed them in (First placed, first filled)

But yes, if you later modify an existing network and create a new chest with a higher priority, newly sorted items will go in there, but existing items won't move over automatically.
